src/ClusterManager/joblog_manager.py

```diff
@@ -38,50 +40,37 @@ def create_log(logdir='/var/log/dlworkspace'):
 
 
 def extract_job_log(job_id, log_path, user_id):
```

Comment:
`extract_job_log` is a duplicate function name as in line 51

Reply:
resolved